MemberExpectations.md

```diff
@@ -0,0 +1,58 @@
+It is important that members of our leadership groups (TSC, CommComm) act
+in a way that not only complies with the code of conduct but that supports
+the vision, mission and operating principles of the Node.js project.
+
+While members have the right and responsibility to exercise independent
+judgment and to express dissenting opinions during discussions within
+the organization, members also have the obligation outside the
+organization (twitter, reddit, etc.) to respect and support decisions
+and values of the majority, even when a member dissented from
+the majority view. A member who does not support a team decision
+or current team value may express the member's opposition within
+the team in an appropriate manner, but must not take actions publicly
+that have the purpose or result of undermining the decisions, values
+or actions of the team. Accordingly, a member who itends to publicly
+oppose a team decision action should resign from the team before doing so.
+
+In addition to being willing to act as a team player, members of our
+leadership groups must conduct themselves in a professional and
+respectful manner. Some general
+guidelines include:
+
+- Serve as ambassadors of the vision, mission and operating
+  principles of the Node.js project.
+- Treat all community members as professionals. with respect, consideration,
+  and valuing a diversity of views and opinions. Strive to avoid preferential
+  treatment, and hold everyone (including ourselves) accountable to the same
+  set of standards.  Everyone gets to speak up. 
+- Deal with issues directly with the person in question. Resist complaining
+  about others in the project in a public sphere.
+- Keep your promises. Your word is your bond. Commit to anything you can
```

Comment:
The second sentence doesn't parse too well for me. `Commit only to things you can deliver upon`?

More generally, I am not sure if this one really fits in here. Is this a statement about deliverables (concrete work)? If so, given that many/most are working on a volunteer basis, this is basically encouraging people to not sign up for anything unless they can guarantee they will come through.

Reply:
I did not read it as meaning concrete deliverables, but I can see how that it could be read this way.  I'll think about it and @hackygolucky might have suggestions for better wording as well.

Reply:
Maybe `If you commit to something, deliver it.`

Reply:
This is not about deliverables though (but I am not entirely what a promise in a non-deliverable sense would mean).